devbuild.sh

```diff
@@ -112,6 +115,8 @@ while :; do
     --app|--app=) usage_error "$1 requires argument." ;;
     --ccpp=?*) CCPP=${1#*=} ;;
     --ccpp|--ccpp=) usage_error "$1 requires argument." ;;
+    --rrfs) RRFS="on" ;;
+    --rrfs=?*|--rrfs=) usage_error "$1 argument ignored." ;;
```

Comment:
The line
```
--rrfs=?*|--rrfs=) usage_error "$1 argument ignored." ;;
```
can apparently be simplified to:
```
--rrfs=*) usage_error "$1 argument ignored." ;;
```
I tried it, and it seems to catch both `--rrfs=` (i.e. nothing after the equal sign) and `--rrfs=abcd` (any non-empty string after the equal sign).  Why not simplify, or am I missing something?  (There are several other occurrences of this pattern in this `case` statement that can also be simplified.)

Reply:
I followed the examples of other options, but am certainly happy to change this to simplify it if that works.

Reply:
I normally do not like to make changes that are completely outside the scope of a PR's purpose.  I'm happy to change the code for the option being added, and then someone (me) can submit a new PR to correct the other ones.

Reply:
Ok, also not a big deal.  Feel free to submit another PR if you have time.  Thanks.